docs/INTEGRATED-DEMO.md

```diff
@@ -25,7 +27,7 @@ dotnet new nugetconfig
 # GITHUB_TOKEN environment variables being present, the token should have 'read:packages'
 # permissions. (replace the \ with ` if using powershell)
 dotnet nuget add source https://nuget.pkg.github.com/rainersigwald/index.json \
-    --name rainer --username '%GITHUB_USERNAME%' --password '%GITHUB_TOKEN%' \
+    --name rainer --username "%GITHUB_USERNAME%" --password "%GITHUB_TOKEN%" \
```

Comment:
this probably needs shell-specific syntax: `cmd` will expand these %-encoded names, while `pwsh` will _if_ they are in double-quotes (which is why they are in single quotes here, because I normally use `pwsh`). The desired end result is that the %-quoted literals are passed through without modification to the nuget.config file.

Reply:
Ah, I see! Then yeah, let's split per shell, cmd/pwsh/bash-ish?

Reply:
Sounds good to me!